Formula/ircii.rb

```diff
@@ -28,8 +28,10 @@ class Ircii < Formula
 
   depends_on "openssl@1.1"
 
+  uses_from_macos "ncurses"
+
   def install
-    ENV.append "LIBS", "-liconv"
+    ENV.append "LIBS", "-liconv" if OS.mac?
     system "./configure", "--prefix=#{prefix}",
                           "--with-default-server=irc.freenode.net",
```

Comment:
Let's go with the more standard
```suggestion
                          "--with-default-server=irc.libera.chat",
```